src/Bridge/RamseyUuid/Serializer/UuidDenormalizer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\RamseyUuid\Serializer;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Serializer\Normalizer\ContextAwareDenormalizerInterface;
+
+class UuidDenormalizer implements ContextAwareDenormalizerInterface
```

Comment:
Why not just `DenormalizerInterface`?

Reply:
Yeah why not :p also fixed 7.1 issue thx.